### PR TITLE
feat: replace ant-d with Material UI

### PR DESCRIPTION
**Pull request by amartya-ws**
### Ticket Link

---

### Related Links

---

### Description
- replaced ant-d with Material UI
- updated tests
---

### Steps to Reproduce / Test

---

---

### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [x] Tests are [changed or added]
- [x] Relevant documentation is changed or added (and PR referenced)

### GIF's

---

### CODE REVIEW

File: app/components/Siderbar/index.tsx [hunk 1]
```diff
@@ -1,11 +1,10 @@
 import React, { useState, useCallback } from 'react';
 import { colors } from '@app/themes';
-import { Button, Drawer, DrawerProps } from 'antd';
+import { Drawer, DrawerProps, IconButton } from '@mui/material';
 import { Link } from 'react-router-dom';
 import styled from '@emotion/styled';
-import { CloseOutlined } from '@ant-design/icons';
 import icon from '@images/ion_rocket-sharp.svg';
-import menuIcon from '@images/menu.svg';
+import MenuIcon from '@mui/icons-material/Menu';
```
Comment (praveenkumar1798) [inline app/components/Siderbar/index.tsx:7]: why is this change ?

File: app/components/Siderbar/index.tsx [hunk 2]
```diff
@@ -1,11 +1,10 @@
 import React, { useState, useCallback } from 'react';
 import { colors } from '@app/themes';
-import { Button, Drawer, DrawerProps } from 'antd';
+import { Drawer, DrawerProps, IconButton } from '@mui/material';
 import { Link } from 'react-router-dom';
 import styled from '@emotion/styled';
-import { CloseOutlined } from '@ant-design/icons';
```
Comment (praveenkumar1798) [inline app/components/Siderbar/index.tsx:6]: replacement in mui ?

File: app/components/LaunchItem/index.tsx [hunk 3]
```diff
@@ -56,12 +56,9 @@ function LaunchItem({ missionName, launchDateUtc, links, id }: Launch) {
         <If condition={!isEmpty(links.wikipedia)}>
           <WikiLink
             data-testid="wiki-link"
-            type="link"
-            rel="noreferrer"
-            target="_blank"
```
Comment (praveenkumar1798) [inline app/components/LaunchItem/index.tsx:61]: Why is this removed ? What is the mui replacement for it ?

File: app/components/Siderbar/index.tsx [hunk 4]
```diff
@@ -63,25 +59,17 @@ const MenuButton = styled(Button)`
...
-`;
-
 const Sidebar: React.FC = () => {
   const [visible, setVisible] = useState(false);
   const { isMobile } = useMedia();
 
   const toggleSidebar = useCallback(() => setVisible((v) => !v), []);
   const sidebarProps: DrawerProps = isMobile
     ? {
-        closeIcon: <CloseOutlined style={{ color: colors.secondary, fontSize: '1.9rem' }} />,
```
Comment (praveenkumar1798) [inline app/components/Siderbar/index.tsx:79]: What is mui replacement ?

File: app/containers/HomeContainer/tests/index.test.tsx [hunk 5]
```diff
@@ -216,25 +216,6 @@ describe('<HomeContainer /> tests', () => {
...
-
-  it('should sort the launches by date in DESC', async () => {
-    const { getByText, getByRole } = renderProvider(
-      <HomeContainer {...defaultProps} launchData={launchData} loading={false} />
-    );
-    fireEvent.mouseDown(getByRole('combobox')!);
-    fireEvent.click(getByText('DESC'));
-    await waitFor(() => expect(history.location.search).toContain('order=desc'));
-  });
-
```
Comment (praveenkumar1798) [inline app/containers/HomeContainer/tests/index.test.tsx:237]: Why is this removed ?

File: app/components/Siderbar/index.tsx (hunk 2 above)
Comment (amartya-ws) [reply app/components/Siderbar/index.tsx:6]: using from '@mui/icons-material/Close';

File: app/components/Siderbar/index.tsx (hunk 4 above)
Comment (amartya-ws) [reply app/components/Siderbar/index.tsx:79]: There is no such prop in mui, please refer latest commit for the changes.

File: app/components/LaunchDetails/tests/__snapshots__/index.test.tsx.snap [hunk 6]
```diff
@@ -4,100 +4,96 @@ exports[`<LaundDetails> tests should render and match the snapshot 1`] = `
...
-              >
-                Name
-              </b>
-               Ship 1
-            </p>
-            <p
-              class="e1uve3jx3 css-139w90b e17qkzd60"
-              data-testid="ship-type"
+            <b
+              class="css-1mq3jm7 e1uve3jx2"
```
[the remark this answers is not in the document]
Comment (amartya-ws) [reply app/components/LaunchDetails/tests/__snapshots__/index.test.tsx.snap]: Updated snapshots.

File: app/containers/HomeContainer/tests/index.test.tsx (hunk 5 above)
Comment (amartya-ws) [reply app/containers/HomeContainer/tests/index.test.tsx:237]: added tests for the select component.

File: app/components/Header/index.tsx [hunk 7]
```diff
@@ -4,26 +4,23 @@
...
 import React from 'react';
-import { Layout } from 'antd';
 import styled from '@emotion/styled';
 import { Link } from 'react-router-dom';
 import { fonts, colors } from '@themes/index';
 import T from '@components/T';
 import logo from '@images/icon-512x512.png';
 
-const StyledHeader = styled(Layout.Header)`
+const StyledHeader = styled.div`
```
Comment (praveenkumar1798) [inline app/components/Header/index.tsx:13]: use AppBar in mui

**Review by praveenkumar1798 — APPROVED** [review]
